packages/lesswrong/lib/collections/typingIndicators/collection.ts

```diff
@@ -0,0 +1,44 @@
+import {ensureIndex} from "../../collectionIndexUtils";
+import {addUniversalFields, getDefaultMutations, getDefaultResolvers} from "../../collectionUtils";
+import {MutationOptions} from "../../vulcan-core/default_mutations";
+import {createCollection} from "../../vulcan-lib";
+import {userOwns} from "../../vulcan-users/permissions";
+import schema from "./schema";
+
+const options: MutationOptions<DbTypingIndicator> = {
+  newCheck: (user: DbUser|null, document: DbTypingIndicator|null) => {
+    if (!user || !document) return false;
+    return true
+  },
+
+  editCheck: (user: DbUser|null, document: DbTypingIndicator|null) => {
+    if (!user || !document) return false;
+    return userOwns(user, document) 
+  },
+
+  removeCheck: (user: DbUser|null, document: DbTypingIndicator|null) => {
+    if (!user || !document) return false;
+    return userOwns(user, document) 
+  },
```

Comment:
I think.. we should probably not grant people the ability to delete typing indicators, even if they own them, and to the extent that we allow this for other collections it's a legacy decision I'd want to move away from